src/Extend/Frontend.php

```diff
@@ -59,6 +60,13 @@ public function route(string $path, string $name, $content = null)
         return $this;
     }
 
+    public function removeRoute(string $name)
```

Comment:
I don't like that this is in both extenders, but I don't suppose we have a choice?

Reply:
well, Frontend adds this type of content routes, I thought it'd makes sense to be able to remove them here as well, especially since the method is always `GET` here, so it needn't be a parameter.

But technically, using the `Routes` extender to remove these routes will still work I believe.